rest_framework_drilldown/views.py

```diff
@@ -86,17 +92,21 @@ def _error(msg='Error'):
             headers['X-Query_Error'] = msg
             return _result(status=400)
 
-        fields = self.request.QUERY_PARAMS.get('fields', [])
+        fields = self.request.query_params.get('fields', [])
         fields = fields and fields.split(',')  # fields parameter is a comma-delimited list
 
         # get parameters that will be used as filters
         filters = {}
-        request_params = self.request.QUERY_PARAMS
+        request_params = self.request.query_params
         for f in request_params:
             if f.split('__')[0] not in self.ignore_fields:   # split so you catch things like "invoice.total__lt=10000"
                 filters[f] = request_params[f]
 
-        qs = self.get_base_query()
+        if 'version' in kwargs:
```

Comment:
You can just do `kwargs.pop('version', None)`

Reply:
Done!